### ISSUE TITLE
Dawn 4 Ubuntu 16.04.4 (vm) build error (master)

### ISSUE THREAD

**Issue by expaand** [EOSIO/eos#2995]
[ 80%] Building CXX object libraries/testing/CMakeFiles/chain_tester.dir/test.cpp.o
/xxx/eos/libraries/testing/test.cpp:4:10: fatal error: 'eosio.token/eosio.token.wast.hpp' file not found
#include <eosio.token/eosio.token.wast.hpp>
         ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
1 error generated.
libraries/testing/CMakeFiles/chain_tester.dir/build.make:62: recipe for target 'libraries/testing/CMakeFiles/chain_tester.dir/test.cpp.o' failed
make[2]: *** [libraries/testing/CMakeFiles/chain_tester.dir/test.cpp.o] Error 1
CMakeFiles/Makefile2:2004: recipe for target 'libraries/testing/CMakeFiles/chain_tester.dir/all' failed
make[1]: *** [libraries/testing/CMakeFiles/chain_tester.dir/all] Error 2
Makefile:160: recipe for target 'all' failed
make: *** [all] Error 2

	>>>>>>>>>>>>>>>>>>>> MAKE building EOSIO has exited with the above error.

**Comment by sschechter** [comment]
I am seeing the same issue

**Comment by nsjames** [comment]
Yup, same thing here. Tried about 4-5 different times nuking the virtual machine each time and starting fresh. Ubuntu 16.04

**Comment by pacificcode** [comment]
@expaand @sschechter @nsjames Can you run the following and send the full output of the build here?
cd /path/to/eos
rm -rf build/
git pull
git submodule update --init --recursive
./eosio_build.sh > eosio_build.log

**Comment by nsjames** [comment]
Thanks to Michael Yeates for the solution. 
Go to `build/contracts` and run `make`. 
Should work fine after.

**Comment by sschechter** [comment]
This worked, much appreciated!

**Comment by robmcc1** [comment]
Thank you for this solution

**Comment by loganwu** [comment]
I encounter the same issue, because eosio.token haven't build, so eosio.token.wast.hpp doesn't exist
